hub.js

```diff
@@ -249,9 +252,14 @@ app.on("upgrade", (req, socket, head) => {
       console.log("[ERROR] Socket error during websocket comm");
     });
     socket.on("close", () => {
-      let container = registry[user].params.container;
-      container.kill((err, data) => {
-        console.log(data);
+      console.log(`Stopping ${user}...`);
+      let entry = registry[user].params.container
+      let container = ishmael.getContainer(entry.id);
+      container.stop((err, data) => {
+        container.remove((err, data) => {
+          ishmael.pruneContainers({"label":user});
+          delete registry[user];
+        });
```

Comment:
This works for individuals closing tabs, but not globally; still causes an `[ERROR] null` from the error message in l. 187, as the socket request is still probably asking for a container that no longer exists. Can we log and ignore?